pytket/conanfile.py

```diff
@@ -32,7 +32,7 @@ def package(self):
         cmake.install()
 
     def requirements(self):
-        self.requires("tket/1.2.26@tket/stable")
+        self.requires("tket/1.2.27@tket/stable")
         self.requires("tklog/0.3.3@tket/stable")
```

Comment:
Do the version of the other packages be updated and the packages uploaded as well?

Reply:
No there are no changes to them; I just rebuilt and uploaded them with their existing versions. TKET's version just needs changing to satisfy the CI check.